engine/src/main/java/com/redhat/cloud/notifications/processors/email/EmailSubscriptionTypeProcessor.java

```diff
@@ -181,7 +178,17 @@ private List<NotificationHistory> sendEmail(Event event, Set<Endpoint> endpoints
         }
 
         if (subject == null || body == null) {
-            return Collections.emptyList();
+            if (subject != null || body != null) {
+                Log.warnf("Only one of the subject and body was found for event: %s/%s/%s", event.getBundleDisplayName(), event.getApplicationDisplayName(), event.getEventTypeDisplayName());
+            }
+
+            if (featureFlipper.isUseDefaultTemplate()) {
+                body = Default.getBody(subject != null, body != null);
+                subject = Default.getTitle();
```

Comment:
Do you plan on keeping `Default` hardcoded as a filesystem-based template after we've enabled the DB templates? 

`Default` would definitely not be a conventional `InstantEmailTemplate` because it would never be associated with any event type, but the relation with the event type is optional so that could still work. It would probably require a new boolean field in `InstantEmailTemplate` to tag it as the default template and prevent linking it with a particular event type.

Reply:
I was thinking on keeping it on the filesystem, but then all the `annotations` required to work and the configuration would need to remain compatible for the filesystem templaltes....
So yes, maybe we will move it to the database as well. We could keep a separate class to do the "custom" configuration 
(i.e. adding the information if a explicit subject or body exists) to the template.